cli/src/cmd/forge/test.rs

```diff
@@ -125,9 +125,13 @@ pub struct TestArgs {
     debug: Option<Regex>,
 
     /// Print a gas report.
-    #[clap(long = "gas-report")]
+    #[clap(long, env = "FORGE_GAS_REPORT")]
     gas_report: bool,
 
+    /// Include the mean and median gas use of fuzz tests in the output.
+    #[clap(long, env = "FORGE_INCLUDE_FUZZ_TEST_GAS")]
+    pub include_fuzz_test_gas: bool,
```

Comment:
will move this to config values as well

Reply:
how'd I go about doing that while still respecting the CLI arg?

Reply:
env always take precedent over config vals, but with bools it's a bit tricky, because if set to true in the config then you can't set it to false via cli easily...

So not sure whether to move this to config at all, perhaps the env var is sufficient?

Reply:
sg, this will require some minor refactoring because this is actually the first config val directly included in the `TestArgs` struct, so I'll rewrite the macro that does all the config magic,
moving this to a followup